src/providers/proxy/proxy_child.c

```diff
@@ -381,14 +374,14 @@ proxy_cli_init(struct pc_ctx *ctx)
         goto done;
     }
 
-    ret = sss_iface_connect_address(ctx, ctx->ev, sbus_cliname, sbus_address,
-                                    NULL, &ctx->conn);
+    ret = sss_iface_connect_address(ctx, ctx->ev, sbus_cliname, SSS_BUS_ADDRESS,
```

Comment:
Why provider uses `sbus_connect_private_send()` and here `sss_iface_connect_address()` is used?

Reply:
In data provider, we created dbus server, so we needed to use asynchronous calls.

sss_iface_connect_address is blocking call, which is ok in init phase.

Reply:
'dp.c::dp_init_send()' still uses `sbus_connect_private_recv()`

Reply:
Now `sbus_connect_private_recv()` is used only within 'sbus_connection_connect.c'.

Maybe it doesn't make sense to keep it public.

Reply:
Yes, it does not make sense anymore. Done.